src/reducers/real.jsx

```diff
@@ -143,18 +143,13 @@ const realReducer = (state = initialState, action) => {
             return deleteContainerState
         }
             
-        case CONTAINER_RENAME: {
-            let containers = state.containers
-            let container = containers.find(ele => ele._id === action.container._id)
+        case CONTAINER_EDIT: {
```

Comment:
for CONTAINER_EDIT, is THIS a new reducer and we are replacing CONTAINER_RENAME?

Reply:
Yes i am renaming the reducer, and i simplified it.

Reply:
this is new action

Reply:
i see. You Changed name from Rename Item to Edit, to be consistent with container menu